test/release/examples/benchmarks/miniMD/helpers/StencilDist.chpl

```diff
@@ -945,14 +945,22 @@ inline
 proc StencilArr.do_dsiAccess(param setter, i: rank*idxType) ref {
   local {
     if myLocArr != nil {
-
-      // return from actual if it's a write or there's no fluff
-      if myLocArr.locDom.member(i) && (setter || zeroTuple(dom.fluff)) then return myLocArr.this(i);
-
-      // return from fluff if it's a read and there's fluff
-      if myLocArr.locDom.myFluff.member(i) && !setter && !zeroTuple(dom.fluff) then return myLocArr.this(i);
```

Comment:
Do we no longer check for `!zeroTuple()` since `myFluff` falls back to `myBlock` if fluff does not exist?